ui-manchette/src/styles/manchette.css

```diff
@@ -1,5 +1,6 @@
 .manchette-container {
   .manchette-actions {
+    z-index: 11;
```

Comment:
Not 100% sure we want to place the menu behind the toolbar… Wouldn't this always hide the menu for the very last waypoint?

![out](https://github.com/user-attachments/assets/0dd2e96d-d593-4f6d-95a3-1d708cd8e2d5)

Reply:
It was a request from @Tguisnet but you're right we will have an issue for the last waypoints !